### PR TITLE
Replace config.get with config.bool

### PR DESCRIPTION
**Pull request by rohansingh**
In the latest Pixlet, `config.get` always returns a string. We instead
need to use `config.bool` to get boolean values.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,38 +9,5 @@ require (
...
-	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/zachomedia/go-bdf v0.0.0-20210522061406-1a147053be95 // indirect
-	go.starlark.net v0.0.0-20211203141949-70c0e40ae128 // indirect
-	golang.org/x/crypto v0.0.0-20210817164053-32db794688a5 // indirect
-	golang.org/x/image v0.0.0-20211028202545-6944b10bf410 // indirect
-	golang.org/x/net v0.0.0-20210825183410-e898025ed96a // indirect
-	golang.org/x/sys v0.0.0-20211205182925-97ca703d548d // indirect
-	golang.org/x/text v0.3.7 // indirect
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	tidbyt.dev/pixlet v0.12.0
```
Comment (betterengineering) [inline go.mod:12]: Huh. Why were all those direct deps there?

**Review by betterengineering — APPROVED** [review]
Short of looking through every toggle - this looks good! 😅

File: go.mod (hunk 1 above)
Comment (rohansingh) [reply go.mod:12]: I looked through every toggle. There's actually only 3 apps that use them.

**Comment by rohansingh** [comment]
I looked through every toggle. There's actually only 3 apps that use them.

File: go.mod (hunk 1 above)
Comment (rohansingh) [reply go.mod:12]: I dunno why they disappeared, but that broke the build. I put them back.

**Pull request merged**